romancal/regtest/test_wfi_pipeline.py

```diff
@@ -0,0 +1,157 @@
+import pytest
+
+from romancal.pipeline.exposure_pipeline import ExposurePipeline
+import roman_datamodels as rdm
+from romancal.assign_wcs.assign_wcs_step import AssignWcsStep
+import numpy as np
+from numpy.testing import assert_allclose
+from .regtestdata import compare_asdf
+import copy
+
+
+def passfail(bool_expr):
+    if bool_expr:
+        return "Pass"
+    else:
+        return "Fail"
+
+@pytest.mark.bigdata
+@pytest.mark.soctests
+def test_level2_image_processing_pipeline(rtdata, ignore_asdf_paths):
+    rtdata.get_data("WFI/image/l1_0001.asdf")
+    rtdata.input = "l1_0001.asdf"
+
+    # Test Pipeline
+    output = "l1_0001_cal.asdf"
+    rtdata.output = output
```

Comment:
Is there a reason you're not using an input file from INS?

Reply:
This is from INS.. I just renamed it to make it simpler. I can revert it to the original, if you wish.

Reply:
I think that would be a good idea. I'm just thinking that when we have to update the tests when INS updates the input files that not having to remember (or reconstruct) name changes will make maintenance more straightforward.

Reply:
Files renamed back to INS names both in the code and in artifactory.